objects/_o-link-complex.scss

```diff
@@ -69,26 +71,14 @@ $o-link-complex-apply-at-breakpoints-for-target:   false !default;
 
   %o-link-complex__target,
   .o-link-complex__target {
-    color: $link-color;
     cursor: pointer;
-
-    &:hover,
-    &:focus {
-      text-decoration: underline;
-      color: $link-color-hover;
-    }
+    @include base-link();
```

Comment:
Me to apparently: https://github.com/chris-pearce/css-guidelines#preprocessors-additional-format-considerations :)

Will fix.